definitions/infra-snmplenovocomputefans/definition.yml

```diff
@@ -0,0 +1,9 @@
+domain: INFRA
```

Comment:
Hi @harrykimpel , we're preferring to use EXT here as a general rule.  Can you update this value here, and the `infra-` prefix in the folder containing these files?

Reply:
alright, makes sense ... will update

Reply:
by the way: are Events already supported at all?

Reply:
@srvaroa